src/index.test.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { ReactNode } from "react";
-import { render, fireEvent, act, screen } from "@testing-library/react";
-import { Props as ReactSearchProps, ReactSearch } from ".";
+import { render, fireEvent, act, screen, waitFor } from "@testing-library/react";
+import { ReactSearch } from ".";
+import { Props as ReactSearchProps } from "./types";
 import * as SearchInput from "./SearchInput";
-import * as SearchResult from "./SearchResult";
+import { SearchResult } from "./SearchResult";
```

Comment:
Nit: Maybe update SearchInput to also use a named export?

Reply:
I'll fix this in a follow-up with all of the issues that came with downgrading RTL.